src/GitHub.TeamFoundation.14/Connect/GitHubConnectSection.cs

```diff
@@ -308,20 +308,20 @@ void HandleCreatedRepo(ILocalRepositoryModel newrepo)
         {
             Guard.ArgumentNotNull(newrepo, nameof(newrepo));
 
-            var msg = string.Format(CultureInfo.CurrentUICulture, Constants.Notification_RepoCreated, newrepo.Name, newrepo.CloneUrl);
-            msg += " " + string.Format(CultureInfo.CurrentUICulture, Constants.Notification_CreateNewProject, newrepo.LocalPath);
+            var msg = String.Format(CultureInfo.CurrentCulture, Constants.Notification_RepoCreated, newrepo.Name, newrepo.CloneUrl);
```

Comment:
Why change `string` to `String` here?

Reply:
I was copy-pasting the culture change and I guess I copy-pasted from somewhere that was uppercase